### PR TITLE
docs(threat-detection): Add threat-detection docs

### PR DESCRIPTION
**Pull request by pedrooot**
### Description

Docs for threat-detection checks are added.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/tutorials/aws/threat-detection.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Threat Detection
+
+Prowler allows you to follow a process of Threat Detection in AWS. To run checks related with Threat Detection use:
```
Comment (jfagoagas) [inline docs/tutorials/aws/threat-detection.md]:
```suggestion
Prowler allows you to do threat detection in AWS based on the CloudTrail log records. To run checks related with threat detection use:
```

File: docs/tutorials/aws/threat-detection.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+```
+prowler aws --category threat-detection
+```
+This comand will run these checks:
+
+* cloudtrail_threat_detection_privilege_escalation
+* cloudtrail_threat_detection_enumeration
```
Comment (jfagoagas) [inline docs/tutorials/aws/threat-detection.md]:
```suggestion
* `cloudtrail_threat_detection_privilege_escalation`
* `cloudtrail_threat_detection_enumeration`
```

File: docs/tutorials/aws/threat-detection.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+*Threat Detection checks will be only executed using `--category threat-detection` flag due to preformance.*
```
Comment (jfagoagas) [inline docs/tutorials/aws/threat-detection.md]: Add this with the `note` format. See other docs.

File: docs/tutorials/aws/threat-detection.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+## Config File
+
+If you want to manage the behavior of Threat Detection checks you can edit `config.yaml` file from `/prowler/config`. In this file you can edit the attributes related with Threat Detection:
```
Comment (jfagoagas) [inline docs/tutorials/aws/threat-detection.md]:
```suggestion
If you want to manage the behavior of the Threat Detection checks you can edit `config.yaml` file from `/prowler/config`. In this file you can edit the following attributes related with Threat Detection:
```

**Review by jfagoagas — CHANGES_REQUESTED** [review]
Really good documentation @pedrooot 👏 

Check my comments when you get a chance, thanks!

File: docs/tutorials/aws/threat-detection.md (hunk 1 above)
Comment (pedrooot) [reply docs/tutorials/aws/threat-detection.md]: Done!

File: docs/tutorials/aws/threat-detection.md (hunk 2 above)
Comment (pedrooot) [reply docs/tutorials/aws/threat-detection.md]: Done!

File: docs/tutorials/aws/threat-detection.md (hunk 3 above)
Comment (pedrooot) [reply docs/tutorials/aws/threat-detection.md]: Done!

File: docs/tutorials/aws/threat-detection.md (hunk 4 above)
Comment (pedrooot) [reply docs/tutorials/aws/threat-detection.md]: Done!

**Review by jfagoagas — APPROVED** [review]
Awesome!

**Pull request merged**